uli_init/simulate.py

```diff
@@ -481,20 +482,28 @@ def __init__(
         expand_factor=5
     ):
         self.molecule = molecule
-        self.para_weight = para_weight
         self.density = density
+        self.para_weight = para_weight
+        self.monomer_sequence = monomer_sequence
+        self.forcefield = forcefield
         self.target_L = None
-        self.remove_hydrogens = remove_hydrogens
         self.epsilon = epsilon
-        self.forcefield = forcefield
+        self.remove_hydrogens = remove_hydrogens
         self.assert_dihedrals = assert_dihedrals
         self.seed = seed
         self.system_mass = 0
         self.para = 0
         self.meta = 0
-        self.type = "melt"
         self.expand_factor = expand_factor
-
+        self.type = "melt"
```

Comment:
right now type is always melt, but you're planning to add more different types, is that right?

Reply:
Not really...this kind of came about due to having 2 different systems classes...right now `System` and `Interface` and is important to handling how the shrinking steps are performed in the `Simulation` class.  So, that's why its hard corded in.  Ultimately I'm going to revamp how different system types are initialized and handled..but for now this is required.